packages/env/lib/env.js

```diff
@@ -219,6 +222,52 @@ module.exports = {
 	ValidationError,
 };
 
+/**
+ * Checks for legacy installs and provides
+ * the user the option to delete them.
+ *
+ * @param {Object} spinner A CLI spinner which indicates progress.
+ */
+async function checkForLegacyInstall( spinner ) {
+	const basename = path.basename( process.cwd() );
+	const installs = [
+		`../${ basename }-wordpress`,
+		`../${ basename }-tests-wordpress`,
```

Comment:
I assume `rimraf` handles the `/` correctly on Windows, but should we output `\` in the confirmation message for Windows users? I suppose it doesn't matter.

Reply:
Is there a convention for this?

Reply:
I have no idea. It's okay. I think `../` is common enough that Windows users running `wp-env` would know what it means 😇

Reply:
Yeah, that's what I think.